.github/workflows/publish-package-docs.yaml

```
@@ -60,11 +60,12 @@ jobs:
           python -m pip install setuptools six wheel
 
       - name: Install project requirements
-        run: |
-          pushd build-directory
-          python -m pip install -r requirements.txt
-          python -m pip install -r docs/docsite/requirements.txt
-          popd
+        run: python -m pip install -r requirements.txt
```

How about combining these?
```suggestion
        run: python -m pip install -r requirements.txt -r docs/docsite/requirements.txt
```

---

personally, I'd keep them separate. The docs/docsite one will change based on which docsite is being published (devel or latest). I'd find it easier to understand that if there is an if/else just wrapped around the docs/docsite one.